MAINTAINERS.adoc

```diff
@@ -103,6 +103,9 @@ If you think some information is outdated, either provide a pull request or send
 |Hibernate ORM with Panache
 |https://github.com/FroMage[Stéphane Épardaud]
 
+|Hibernate ORM with Panache for Kotlin
+|https://github.com/evanchooy[Justin Lee]
```

Comment:
```suggestion
|https://github.com/evanchooly[Justin Lee]
```

Reply:
Ah good catch, maybe @evanchooly didn't want to be found :smile:

Reply:
Ha ha maybe `evanchooy` will know :-P
 
Noticed it in the earlier PR but it was already merged.